zerver/lib/bugdown/api_arguments_table_generator.py

```diff
@@ -117,11 +117,18 @@ def render_table(self, arguments: List[Dict[str, Any]]) -> List[str]:
 
             # TODO: OpenAPI allows indicating where the argument goes
             # (path, querystring, form data...).  We should document this detail.
+            example = ""
+            if 'example' in argument:
+                example = escape_html(json.dumps(argument['example']))
+            else:
+                content_type = list(argument['content'].keys())[0]
+                example = escape_html(json.dumps(argument['content'][content_type]['example']))
```

Comment:
We should hard-code `application/json` as the content type to look up. If it’s ever not `application/json` in the future, we’ll need something other than `json.dumps`; it would be better to get an error here so we know that this needs to be updated.

Reply:
Hardcoded `application/json` and also added an assert statement,